tests/test_ilastik/widgets/test_ImageFileDialog.py

```diff
@@ -23,7 +23,7 @@ def tmp_preferences(tmp_path) -> pathlib.Path:
 def image(tmp_path) -> Path:
     image_path = Path(tmp_path) / "some_picture.png"
     image_path.touch()
-    return image_path
+    return image_path.resolve()
```

Comment:
Leftover code change?.. This doesn't seem to be build-related.

Reply:
it's test related :o. On windows runners on gitlab ci, the username is so long that tmp_path returns a "short name" (windows stuff... like `BLAH~1` instead of `blah-my-long-name`). ImageFileDialog seems to resolve those internally..

Reply:
Are 8.3 filename restrictions still relevant these days?..

Reply:
yeah :/